prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-toa-maintenance/enrollee-toa-maintenance.component.html

```diff
@@ -0,0 +1,45 @@
+<app-page mode="full"
+          [showToolbar]="true"
+          (back)="onBack()">
+  <app-page-header>TOA Maintenance</app-page-header>
+  <app-page-subheader *ngIf="!previewingToa">
+    <ng-container appPageSubheaderTitle>View TOA</ng-container>
+    <ng-container appPageSubheaderSummary>
+      Select what TOA you would like to view
+    </ng-container>
+  </app-page-subheader>
+
+  <div class="row"
```

Comment:
You won't want to use the `[hidden]` binding since it doesn't remove the list markup from the DOM, and instead you'll want to use ngIfElse and templates.

Feels like you'll want to use `*ngIf="!previewingToa; else preview" `, and then 

```
<ng-container *ngIf="!previewingToa; else preview">
  <div class="row">
    <!-- List content -->
  </div>
</ng-container>

<ng-template *ngIf="previewingToa">
  <div class="row">
    <!-- Preview -->
  </div>
</ng-template>
```

Reply:
Somehow using ngIf triggers the observable async http call when set previewingToa=null